ambari-server/src/main/java/org/apache/ambari/server/actionmanager/ExecutionCommandWrapper.java

```diff
@@ -162,19 +156,10 @@ public ExecutionCommand getExecutionCommand() {
             "While scheduling task {} on cluster {}, configurations are being refreshed using desired configurations of {}",
             executionCommand.getTaskId(), cluster.getClusterName(), desiredConfigs);
 
-        // then clear out any existing configurations so that all of the new
-        // configurations are forcefully applied
-        configurations.clear();
-        executionCommand.setConfigurationTags(configurationTags);
+        configurations = configHelper.getEffectiveConfigProperties(cluster, configurationTags);
```

Comment:
Is this a major bug in 2.7.0 ?

Reply:
What I was pointing out here is the fact that we are now making a call to get effective tags (including config groups), did we somehow break config groups since 2.7?

Reply:
Please, note that this is done only when overrideConfigs flag is set.
This is only expected when agent is not mapped to cluster and agent config cache is empty (e.g. we need to send configs with command).
Normal flow is not affected.